photon-client/src/components/dashboard/tabs/ObjectDetectionTab.vue

```diff
@@ -31,6 +32,14 @@ const interactiveCols = computed(() =>
 
 <template>
   <div v-if="currentPipelineSettings.pipelineType === PipelineType.ObjectDetection">
+    <pv-select
+      v-model="currentPipelineSettings.model"
```

Comment:
So pretty sure that the v-model here is of type int, but you've said that pipelinesettings.model is a string.